bad_cert_demo3.js

```diff
@@ -0,0 +1,24 @@
+var tls = require('tls');
+var https = require('https');
+
+var socket3 = tls.connect({
+    port: 1337,
+    host: 'https://example1.com',
+    rejectUnauthorized: true,
+    checkServerIdentity: function(servername, peer) {  
```

Comment:
Coverity found issue: TLS uses bad certificate verification - CWE-295, Medium Severity

**Improper certificate verification may allow an attacker to spoof a trusted entity by using a man-in-the-middle attack.**

**How to fix:** Use the default implementation either implicitly or explicitly, as the default function provided by the "tls" module is secure. When using a custom implementation, make sure that the function throws an error when the certificate does not match, and not just returns "undefined" on every input.
<!-- Coverity 055c1a5f38b3326bf18c0ea5b9fcef43 -->